.travis.yml

```diff
@@ -26,8 +26,9 @@ install:
   - sudo pip install awscli --upgrade
 script:
   - ./test.sh
-  - docker run -a stdout -a stderr --net host --rm --privileged --name build-machine -v "$(pwd)":/go/src/github.com/kuzzleio/sdk-go kuzzleio/sdk-cross:amd64 /build.sh
   - sh .codepipeline/start_kuzzle.sh
+  - docker run --rm -it --mount type=bind,source="$(pwd)",target=/go/src/github.com/kuzzleio/sdk-go kuzzleio/sdk-cross:android-x86 /build.sh
```

Comment:
Replace the old mount syntax `--mount type=bind,source="$(pwd)",target=/go/src/github.com/kuzzleio/sdk-go` by `-v "$(pwd)":/go/src/github.com/kuzzleio/sdk-go`
(Everywhere)